pyproject.toml

```diff
@@ -38,6 +38,7 @@ dependencies = [
   "sortedcontainers",
   "protobuf>=3.20, <5.0",
   "ddsketch",
+  "nest-asyncio",
```

Comment:
`nest-asyncio` is used here, it patches `asyncio` in order to run async code while an event loop is running. The main place this happens is inside a notebook. If we want, I can probably rework this to be a context manager to prevent side-effects

https://github.com/Arize-ai/phoenix/pull/1754/files#diff-92ab90fe4b323d7c744091d4218e80f4c162b68bb2883454cb07bf0bceba4dfbR108-R109

Reply:
If memory serves me right, it caused some issues with uvicorn - I can test out this path.

Reply:
I tried a few strategies but it doesn't look like there's an easy way to only temporarily apply this patch

Reply:
using anyio blocking portals to avoid `nest-asyncio`